### ISSUE TITLE
Support for TheRipper93's Real Dice

### ISSUE THREAD

**Issue by FlorianH1989** [chrisk123999/chris-premades#182]
[CPR-Troubleshoot.txt](https://github.com/chrisk123999/chris-premades/files/14071966/CPR-Troubleshoot.txt)

One of my players is currently playing a hexblade warlock. She is using the hex spell and the hexblade curse in combination and gets to add her proficiency modifier for damage rolls and 1d6 necrotic damage. All of my players are using TheRipper93's Real Dice module so they can roll manually.

My player gets multiple prompts for different stages of the damage roll. In this case it looks like this:
1. 1d8 + 4
2. 1d8 + 4 + 2
3. 1d8 + 4 + 2 + 1d6

She enters her dice rolls and only the last prompt is calculated. So the math works and she gets all of her bonuses. The damage of the first two prompts are neglected.

Is there a way to display only the correct prompt?

I know there is currently an option for manually rolling CPR, but my players want to enter the value themselves. It's part of the fun for them.

**Comment by chrisk123999** [comment]
Unfortunately this means that the module is hooking all dice rolls, I'm not sure this is easily solvable.

**Comment by FlorianH1989** [comment]
Thank you for the fast reply! 
Okay, maybe I'll better ask TheRipper93 then. If you have any ideas let me know!

**Comment by FlorianH1989** [comment]
Okay, here is the issue I opened at Ripper's GitHub: https://github.com/theripper93/real-dice/issues/4

He cannot reproduce the double prompt when adding a damage bonus as effect. 
I just tested for myself. When I create a new effect with an extra 1d10 as damage bonus, then there is no double prompt. I used **system.bonuses.All-Damage** for this example. 

Can CPR calculate a bonus damage differently?


<img width="899" alt="Bildschirm­foto 2024-01-27 um 21 41 21" src="https://github.com/chrisk123999/chris-premades/assets/110769699/0565cda5-1783-4abe-a88d-ba215e2870f4">
